content/strategy-goals/strategy/code-graph/search/core.md

```diff
@@ -12,22 +12,27 @@ Quicklinks:
 
 ## What's next and why
 
-### FY22Q4
+### FY23Q1
 
 #### Goals
 
-- Make Sourcegraph Search work well with large-scale monorepos (time to first result for unindexed search on monorepos up to 60GB is < 10s and <2s for indexed, and time to indexing is always <30 minutes).
-- Sourcegraph Cloud indexes public repositories globally from 4 non-GitHub code hosts.
+- **Monorepo performance**: At a P75 level, a synthetic repo of 15GB will index in less than 30 minutes, indexed searches complete in < 2s and unindexed searches complete in < 10s
+- **Ranking**: We start tracking ranking quality, using selected search results as a proxy.
+- **Code host coverage**: Sourcegraph Cloud indexes public repositories globally from the most popular package hosts.
 
 #### Details
 
-**Monorepo performance** is a recurrent pain point for large Enterprise customers. Unindexed monorepo performance is still poor, and several facets of search on large monorepos cause significant load on gitserver.
+**Monorepo performance**: is a recurrent pain point for large enterprise customers. Having replicated large monorepo setups, and identified the unindexed monorepo performance is still poor, and several facets of search on large monorepos cause significant load on gitserver.
+
+**Ranking**: As a first step towards improving the ranking of our search results, we will start tracking the quality of search results, using the index of user-selected results as a proxy metric. Having this tracking in place will help measure the success of future improvements, and drive the areas of ranking we choose to focus on.
 
-**Code host coverage**: we've chosen to make progress towards [indexing the entire OSS universe](https://about.sourcegraph.com/blog/why-index-the-oss-universe/) by indexing repositories from different code hosts. By focusing on code host coverage, we strive to make Sourcegraph.com a truly universal code search engine: Sourcegraph will not be limited to searching over just the most popular code hosts thus increasing the discoverability and visibility of OSS code regardless of the code host. We see as a better increment towards making sourcegraph.com a compelling OSS code search engine than our previous goal of scaling the global index to 5.5M repositories (see "what we're not working on" below).
+**Code host coverage**: as we seek to keep expanding the code we index to include more non-GitHub.com or GitLab.com code hosts, we will add support for package host integrations (PyPi, Rubygems, NuGet, Crates, proxy.golang.org). This will not only increase our code host coverage, but also be a stepping stone towards unblocking [use cases based on the dependency graph of repositories](https://docs.google.com/document/d/1SkM8CG0IksvPEKRBRVLKipiRJTopx6Vq_hSWRJ9NyKs/edit#).
 
 #### What we're not working on and why
```

Comment:
The scaling sourcegraph to 5.5M has been there for a few months now and the explanation can probably be summarized further.